modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc

```diff
@@ -5,6 +5,12 @@
 
 This topic lists new features added in Redpanda Cloud.
 
+== May 2024
+
+=== Namespaces renamed resource groups
+
+As part of Redpanda Cloud's simplified navigation, namespaces now are called glossterm:resource group[,resource groups]. The functionality is the same, but now the *Clusters* page lists the clusters in all of your resource groups, and the *Networks* page lists the networks in all of your resource groups. 
```

Comment:
I think we can put more meat on this. We should be talking about default resource groups and how we have flattened the hierarchy that used to exist previously

Reply:
@towfiqa: If you hover over the term **resource groups** here, you see the text for the glossary entry. What should we add to this?
![image](https://github.com/redpanda-data/docs/assets/107008679/fe0c097a-3dee-4404-b077-c9d95d874fd0)